idaes/surrogate/pysmo/polynomial_regression.py

```diff
@@ -949,20 +949,18 @@ def results_generation(self, beta, order):
         """
         This function prints the results of the fitting to the screen.
         """
-        results_df = pd.Series()
+        results_df = pd.Series(dtype='float64')
```

Comment:
This is more of a general observation, but do we need `results_df` to be a DataFrame while we collect the results? From what I understand in the function, it should be possible to use a simple list to accumulate each item as a `pd.Series` (or even as the underlying `dict` literal), and then convert the list of `pd.Series`/`dict` to a DataFrame at the end:

```py
result_items = []
for i in range(...):
    col_name = f"..."
    result_items.append({col_name: beta[i, 0]})
return pd.DataFrame(result_items)
```

Note that I might have misunderstood the expected shape of the dataframe (I can never picture what `pd.concat()` does for all possible inputs, see the comment below), in which case I'd recommend to add a comment or a docstring that shows what's the expected format of the dataframe (e.g. just call the function manually and copy `df.head(5)` in the docstring)